apps/meteor/app/push/server/push.ts

```diff
@@ -138,6 +189,33 @@ class PushClass {
 		}
 	}
 
+	private async getFcmAuthorizationToken(): Promise<{ token: string; projectNumber: string }> {
+		const credentialsString = settings.get('Push_google_api_credentials');
+		if (!credentialsString) {
+			throw new Error('Push_google_api_credentials is not set');
+		}
+
+		const credentials = JSON.parse(credentialsString as string) as FCMCredentials;
```

Comment:
Since the string comes from a setting, we should validate that the resulting object actually matches the FCMCredentials type.

Reply:
should I use ajv or Match for that?

Reply:
did it using ajv